### ISSUE TITLE
Add typescript Support

### ISSUE THREAD

**Issue by Andreas-Schoenefeldt** [garbit/lottie-web-vue#10]
If I try to use the library in a typescript project I get the following compilation error:

```
TS7016: Could not find a declaration file for module 'lottie-web-vue'. './node_modules/lottie-web-vue/dist/lottie-web-vue.umd.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/lottie-web-vue` if it exists or add a new declaration (.d.ts) file containing `declare module 'lottie-web-vue';`
```

My `main.ts` (vue 3):

```js
import { createApp } from 'vue'
import LottieAnimation from 'lottie-web-vue'
// ...

createApp(App).use(LottieAnimation);
```

It would be great, if you could adopt typescript support for the library :)

**Comment by Andreas-Schoenefeldt** [comment]
It turned out there is a pretty simple solution :), thx to https://blog.kewah.com/2018/typescript-declaration-file/

in the `tsconfig.json`:

```json
{
  "compilerOptions": {
    "noImplicitAny": false
  }
}
```

I leave the ticket open. It would be great to have the d.ts file for this lib eventually.

**Comment by garbit** [comment]
Hi @Andreas-Schoenefeldt - I've been working on a new Typescript declared plugin that has just been released (v2.0.0). I'm testing it now and it's been entirely rewritten to support this ticket.

**Comment by Andreas-Schoenefeldt** [comment]
Great news 🥳
